features/checkout/paying_for_order/selecting_order_payment_method.feature

```diff
@@ -0,0 +1,21 @@
+@checkout
+Feature: Selecting an order payment method
+    In order to pay for my order
+    As a Customer
+    I want to be able to choose a payment method
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "PHP T-Shirt" priced at "$19.99"
+        And the store ships everywhere for free
+        And the store allows paying with "Offline"
```

Comment:
Maybe we could use something like "Cash on Delivery"? Offline is a bit too technical.